thread/thread.h

```diff
@@ -61,8 +65,10 @@ namespace photon
     const uint64_t DEFAULT_STACK_SIZE = 8 * 1024 * 1024;
     // Thread stack size should be at least 16KB. The thread struct located at stack bottom,
     // and the mprotect page is located at stack top-end.
+    // reserved_space must be <= stack_size / 2
     thread* thread_create(thread_entry start, void* arg,
```

Comment:
thread_create now able to set THREAD_ENALBE_WORK_STEALING when creating, but still need methods to enable/disable stealing for thread temporary.